lib/freight/conf.sh

```diff
@@ -6,18 +8,23 @@ VARLIB="/var/lib/freight"
 VARCACHE="/var/cache/freight"
 
 # Default architectures.
+# shellcheck disable=SC2034
```

Comment:
Ah OK, I was reading the wiki here (https://github.com/koalaman/shellcheck/wiki/Directive), perhaps it's not in the version we're using or similar.